.golangci.yaml

```diff
@@ -0,0 +1,19 @@
+run:
+    timeout: 2m
+linters:
```

Comment:
That is a LOT of linters!

Reply:
Yeah, it's quite a collection. Most are very narrowly focused.